src/pm/hydra/tools/bootstrap/external/slurm_query_node_list.c

```diff
@@ -75,7 +75,10 @@ static HYD_status list_to_nodes(char *str)
 
     string = MPL_strdup(str);
 
-    /* compile regex patterns for nodelist matching */
+    /* compile regex patterns for nodelist matching
+     * see https://tools.ietf.org/html/rfc952 and
+     * https://tools.ietf.org/html/rfc1123 for more details
+     * on hostnames format */
 
```

Comment:
Why not simply paste the text that you quoted in the issue comment? No one would read the rfc for details :).

Reply:
ok

Reply:
Feel free paraphrase and make the quote shorter. We just need the gist in the comment.